### ISSUE TITLE
[A11Y] Missing bypass block link

### ISSUE THREAD

**Issue by foxbunny** [indico/indico#5932]
**Describe the bug**

The pages are missing a bypass block link (the "Skip to content").

**To Reproduce**

1. Go to any page
2. Press Tab to navigate
3. Observe that the first focused item is not a bypass block link

**Expected behavior**

Expected the first focused item to be a link to the main content.

**Additional context**

- https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks.html
- https://talk.getindico.io/t/navigation-bypass-blocks/3203
